File: TP/TP2/Code/1A.cpp
/* Retourner le maximum de deux réels */
#include <iostream>

/* Fonction */

float max_ret(float a, float b)
{
  if (a > b)
    return a;
  else
    return b;
}

/* Utilisation */

int main()
{
  float a;
  float b;

  std::cout << "Donne moi une valeur : " << std::flush;
  std::cin >> a;
  std::cout << "Donne moi une autre valeur : " << std::flush;
  std::cin >> b;

  std::cout << "La valeur la plus grande est : " << max_ret(a, b) << std::endl;

  return 0;
}

File: TP/TP2/Code/Python/1A.py
#!/usr/bin/python
# Retourner le maximum de deux réels


def return_max(foo, bar):
    if foo > bar:
        return foo
    else:
        return bar


print('Donne moi une valeur : ', end='')
foo = float(input())
print('Donne moi une autre valeur : ', end='')
bar = float(input())

print(return_max(foo, bar))  # Debug
